menu.json

```diff
@@ -2,6 +2,7 @@
   "BEEP BOOP",
   "MEET PIPE",
   "INPUT OUTPUT",
+  "READ IT",
```

Comment:
I don't totally like the exercise name. If anyone have another better name is more than welcome